static/js/components/material/ValidatedDialog.js

```diff
@@ -0,0 +1,119 @@
+// @flow
```

Comment:
What's different about `ValidatedDialog` compared to `Dialog`? It looks like there isn't much change if I diff the two components. Is it possible to modify `Dialog` instead to support the validation functionality?

Reply:
Differences in how onAccept, onCancel events are dealt with among other things, but I figured out a way to make Dialog handle both situations where validation is and isn't required.